source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -184,7 +184,7 @@ In order to use a mop-up job, you need:
 
 The following UML sequence diagram shows an example of an incomplete payment journey, and how a mop-up job would clean it up:
 
-![A mop-up job acting on an incomplete payment journey](/images/incomplete-payment-journey-sequence.svg)
```

Comment:
Can we remove the file too from images/ https://github.com/alphagov/pay-tech-docs/blob/master/source/images/incomplete-payment-journey-sequence.svg

Reply:
@kbottla I've got a separate ticket to remove all of the unused images from this repo - there's about 20 of them.

I'll remove this image in that piece of work.